tests/phpunit/tests/actions.php

```diff
@@ -416,6 +469,8 @@ function test_remove_anonymous_callback() {
 	 * Test the ArrayAccess methods of WP_Hook
 	 *
 	 * @ticket 17817
+	 *
+	 * @covers ::has_action
```

Comment:
```suggestion
 	 * @covers WP_Hook::offsetGet
	 * @covers WP_Hook::offsetSet
 	 * @covers WP_Hook::offsetUnset
```

This test is testing `ArrayAccess`, i.e. being able to use the instance of `WP_Hook` (which is stored in the global `$wp_filter`) as an array.